contracts/cw721-fixed-price/src/contract.rs

```diff
@@ -160,16 +160,19 @@ pub fn execute_receive(
         extension: config.extension.clone(),
     });
 
-    let callback = CosmosMsg::Wasm(WasmMsg::Execute {
-        contract_addr: config.cw20_address.to_string(),
-        msg: to_binary(&mint_msg)?,
-        funds: vec![],
-    });
+    if let Some(cw721) = &config.cw721_address {
```

Comment:
You can use `.ok_or(Err)` here to simplify code

Reply:
Had a try, not sure if it's more idiomatic / cleaner, I'll leave it as it is here.